src/org/pahappa/systems/ticketing/views/TicketView.java

```diff
@@ -20,52 +25,130 @@ public void displayMenu() {
         System.out.println("********* Call Center Ticket System *********\n\n");
         boolean running = true;
         while (running) {
-            System.out.println("Choose an operation:");
-            System.out.println("1. Create Ticket");
-            System.out.println("2. Get All Tickets");
-            System.out.println("3. Get Tickets of Status");
-            System.out.println("4. Update Ticket");
-            System.out.println("5. Delete Ticket");
-            System.out.println("6. Exit");
-            System.out.println();
-
-            int choice = scanner.nextInt();
-            scanner.nextLine(); // Consume the newline character
-
-            switch (choice) {
-                case 1:
-                    createTicket();
-                    break;
-                case 2:
-                    getAllTickets();
-                    break;
-                case 3:
-                    getTicketsOfStatus();
-                    break;
-                case 4:
-                    updateTicket();
-                    break;
-                case 5:
-                    deleteTicket();
-                    break;
-                case 6:
-                    running = false;
-                    break;
-                default:
-                    System.out.println("Invalid choice. Please try again.");
+            try {
+                System.out.println("Choose an operation:");
+                System.out.println("1. Create Ticket");
+                System.out.println("2. Get All Tickets");
+                System.out.println("3. Get Tickets of Status");
+                System.out.println("4. Update Ticket");
+                System.out.println("5. Delete Ticket");
+                System.out.println("6. Exit");
+                System.out.println();
+
+                int choice = scanner.nextInt();
+                scanner.nextLine(); // Consume the newline character
+
+                switch (choice) {
+                    case 1:
+                        createTicket();
+                        break;
+                    case 2:
+                        getAllTickets();
+                        break;
+                    case 3:
+                        getTicketsOfStatus();
+                        break;
+                    case 4:
+                        updateTicket();
+                        break;
+                    case 5:
+                        deleteTicket();
+                        break;
+                    case 6:
+                        running = false;
+                        break;
+                    default:
+                        System.out.println("Invalid choice. Please try again.");
+
+                }
+            } catch (java.util.InputMismatchException e) {
+                System.out.println("Invalid input. Please try again.");
+                scanner.nextLine();
 
             }
+
         }
     }
 
+    TicketServiceImpl objService = new TicketServiceImpl();
```

Comment:
Using meaningful name such as `TicketService ticketService = new TicketServiceImpl();` It becomes easier to remember the name of this object, since it's related to what you're trying to do. As compared to objService